### ISSUE TITLE
Instance Cross Entropy for Deep Metric Learning and its application in SimCLR-A Simple Framework for Contrastive Learning of Visual Representations

### ISSUE THREAD

**Issue by XinshaoAmosWang** [XinshaoAmosWang/Ranked-List-Loss-for-DML#5]
https://xinshaoamoswang.github.io/paperlists/2020-02-16-arXiv/#foundation-of-deep-learning

**Comment by yutaizhou** [comment]
Hello! I was wondering if the code for your ICE paper has been released? And whether it would be compatible with a framework like [Pytorch Metric Learning](https://kevinmusgrave.github.io/pytorch-metric-learning/)
